Formula/pysvn.rb

```diff
@@ -0,0 +1,111 @@
+class Pysvn < Formula
+  desc "Python interface to Subversion"
+  homepage "http://pysvn.tigris.org/"
+  url "http://pysvn.barrys-emacs.org/source_kits/pysvn-1.8.0.tar.gz"
+  sha256 "39596f4884ed689cdb5a4e210e421724302a566c7ba756cc4d46bbfeb0c8326b"
+
+  option "without-python", "Build without python2 support"
+
+  depends_on :python => :recommended if MacOS.version <= :snow_leopard
+  depends_on :python3 => :recommended
+  depends_on "openssl"
+  depends_on "subversion"
+
+  # Fix configure to respect CPPFLAGS, CXXFLAGS and LDFLAGS
+  # http://pysvn.tigris.org/issues/show_bug.cgi?id=206
+  patch :DATA
```

Comment:
I'd like to see upstream merge this before we include it given it's non-trivial and a good indication of how maintained this project is. Sorry for the delay.